src/main/java/com/auth0/client/mgmt/filter/CheckpointPaginationFilter.java

```diff
@@ -0,0 +1,38 @@
+package com.auth0.client.mgmt.filter;
+
+public class CheckpointPaginationFilter extends BaseFilter {
```

Comment:
We can use PageFilter for now instead of creating new class CheckpointPaginationFilter